articles/mysql/flexible-server/concepts-server-parameters.md

```diff
@@ -48,7 +48,7 @@ Review the [MySQL documentation](https://dev.mysql.com/doc/refman/5.7/en/innodb-
 |Burstable (B1s)|1|1|134217728|33554432|134217728|
 |Burstable (B1ms)|1|2|536870912|134217728|536870912|
 |Burstable|2|4|2147483648|134217728|2147483648|
-|General Purpose|2|8|6442450944|134217728|6442450944|
+|General Purpose|2|8|5368709120|134217728|5368709120|
```

Comment:
For GP V core 2 there was changes made from backend, so document need to update as well. Approving the changes